src/main/java/io/storj/codeowners/Config.java

```diff
@@ -17,13 +18,17 @@ public class Config {
 
     public static final Pattern REVIEWER_COUNT_PATTERN = Pattern.compile("#\\s*gerrit-codeowners.reviewer-count:\\s*(\\d+)\\s*");
 
+    public static final Pattern USE_GIT_HISTORY = Pattern.compile("#\\s*gerrit-codeowners.use-git-history: true");
```

Comment:
this is funny config parsing

Reply:
Yeah, kind of. I think the original motivation to use optional features on top of existing, github compatible CODEOWNERS file....